ONSdigital/eq-questionnaire-runner
# Impute unrelated relationships

### What is the context of this PR?
Updates relationships in the relationship store when an answer is provided for the "Related to anyone else?" question:
- If `Yes` is selected, any `Unrelated` relationships that remain for that person are removed. Note that other relationships (i.e. not `Unrelated` are left intact)
- If `No` is selected, the remaining relationships for that person are added to the relationship store with a relationship of `Unrelated`

### How to review 
- Use the `test_relationships_unrelated` schema
- Try different routes to test the behaviour
- Check that the appropriate relationships are stored (use `/dump/debug`)
- Check that the submission payload is as expected (use `/dump/submission`)

### Checklist
